pkg/ext-proc/backend/vllm/metrics.go

```diff
@@ -77,6 +78,9 @@ func promToPodMetrics(metricFamilies map[string]*dto.MetricFamily, existing *bac
 	if err == nil {
 		updated.KVCacheUsagePercent = cachePercent.GetGauge().GetValue()
 	}
```

Comment:
Pls also update the MaxActiveLoras and close https://github.com/kubernetes-sigs/llm-instance-gateway/issues/22

For some reason I don't see that TODO here, maybe you need to rebase?